_posts/2019-12-03-escaping-the-tar-pit-at-rubyconf.markdown

```diff
@@ -0,0 +1,164 @@
+---
+layout: post
+title: "Escaping The Tar Pit at RubyConf 2019"
+date: 2019-12-03 10:00:00
+categories: ["quality"]
+author: etagwerker
+---
+
+This year I had the honor to speak at [RubyConf](https://rubyconf.org) in Nashville.
+It was my second time attending the conference and first time as a speaker. I
+talked about `skunk`, a library to [calculate the StinkScore](https://github.com/fastruby/skunk)
+of a module or set of modules.
+
```

Comment:
@etagwerker hm, don't you want to add a link to your talk or maybe embed the video somewhere in the article?

Reply:
@lubc haha, yeah! I started the article with that in mind and then totally forgot about it... 👍